src/02_evaluate_better_models.R

```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
                                              parametrosModelo = tabla_coeficientes_auxiliar)
   
   readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
-                     path =  "data/processed/parametros_modelo_logistico_laal_ig.json")
+                     path = "data/processed/parametros_modelo_logistico_laal_ig.json")
   ModeloDimorfismoAlbatros <- ModeloDimorfismo$new()
   ModeloDimorfismoAlbatros$loadParameters("data/processed/parametros_modelo_logistico_laal_ig.json")
   
   prob <- ModeloDimorfismoAlbatros$predict(Datos)
   y_test <- ifelse(Datos$sexo == 'M', 1, 0)
   datos_roc <- data.frame(y_test, prob)
-  errores <- append(errores, calculadorROC$calculateError(datos_roc,umbral))
+  errores <- append(errores, calculadorROC$calculateError(datos_roc, umbral))
 }
 
 es_error_minimo <- errores == min(errores)
 write_csv(tabla_importada[es_error_minimo,], paste0(ruta_resultados,'tabla_mejores_modelos.csv'))
 
-readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), path =  "data/processed/parametros_mejor_modelo_logistico_laal_ig.json")
\ No newline at end of file
+readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), path = "data/processed/parametros_mejor_modelo_logistico_laal_ig.json")
```

Comment:
El mismo comentario que en la linea 29:
```suggestion
readr::write_lines(
    jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
    path = "data/processed/parametros_mejor_modelo_logistico_laal_ig.json"
)
```